client/signup/steps/plans/index.jsx

```diff
@@ -114,40 +115,66 @@ export class PlansStep extends Component {
 			planTypes,
 			flowName,
 			showTreatmentPlansReorderTest,
+			// eslint-disable-next-line no-unused-vars
 			isLoadingExperiment,
 			isInVerticalScrollingPlansExperiment,
 			isReskinned,
 		} = this.props;
 
+		const loadingPlanDisplay = (
+			<div className="plans__loading-container">
+				<PulsingDot delay={ 400 } active />
+			</div>
+		);
+
+		const treatmentPlanDisplay = (
```

Comment:
Loading this along side the default component has become too big:
![image](https://user-images.githubusercontent.com/1842898/134711996-dd1db288-ee4c-4e23-a34c-a3ed3b8b1d65.png)

Let's load this in async; here is a good example of how to do it: https://github.com/Automattic/wp-calypso/blob/trunk/client/me/purchases/purchases-site/index.jsx#L38 . We can also consider to load the default component in async as well so the treatment group won't have to load it, but I'll defer that to you :)

Reply:
Should be loading async now.

Reply:
Yep. Looks so great :)
![image](https://user-images.githubusercontent.com/1842898/135003585-8cf36757-76e3-42d3-8a07-ca4eb2c0e51f.png)